docs/understanding-radicle/how-radicle-works.md

```diff
@@ -0,0 +1,316 @@
+Radicle, your identity is a *portable* JSON document that defines a user or a project on the Radicle network.
+
+The most important part is the hashed content identifier—the **peer ID**—which identifies it uniquely wherever it's
+stored or discovered on the Radicle network. Each Radicle identity also has a few required metadata fields, which
+determine whether it's a user or project identity:
+
+```rust
+struct User {
+    /// A short name (nickname, handle), without any prefix such as the `@`
+    /// character
+    name: String,
+}
+
+struct Project {
+    /// A short name
+    name: String,
+
+    /// A slightly longer description (should fit in a headline)
+    description: Option<String>,
+
+    /// The default branch. "master" is assumed for git repositories if
+    /// unspecified.
+    default_branch: Option<String>,
+}
+```
+
+An identity can also be extended with additional metadata as needed by the client or interface that's interacting with
+it.
+
+When you create a user identity, it's signed with an Ed25519 keypair that ensures only you can change the identity and
+push changes to the Radicle network on behalf of your peer ID.
+
+When you create a project, your identity is added as a **delegate**—more on those in a moment.
+
+### Why Radicle starts with identity
+
+At their core, Git repositories are just commit histories. Eventually, these commit histories develop into complex trees
+of folders, files, and code.
+
+But if you take two Git repositories at face value, there's no genuine way of comparing them. You can look at the commit
+logs and diff individual files, but there's no way of knowing, just through the data that is stored in Git, whether
+project A and project B are meant to converge someday. Are their histories and `HEAD` meant to be the same? Can you tell which repository is meant to be canonical?
+
+All this uncertainty happens because source code is *portable*—it can be cloned, edited, and hosted somewhere else
+without the project's maintainer knowing about it.
+
+Centralized code collaboration platforms like GitHub and GitLab create also create identities for the projects they
+host. If you create two identical repositories on these platforms, their identities are `github.com/bob/project` and
+`gitlab.com/bob/project`. You might know they're the same, but to someone else, there's no guarantee that the projects
+are the same despite seemingly sharing a `bob/project` identity.
+
+With centralized platforms, the code remains portable, but the *identity is non-portable*.
+
+A portable Radicle identity solves this problem because you always own your Radicle identity. Your identity might be replicated across the Radicle network, but because your key signs it, it's always yours. Your project may be replicated across the Radicle network, but because your identity is a delegate, only you can change the canonical remote/branch.
```

Comment:
Not sure we have really solved the problem of what is canonical within a project scope yet. In the end it still might be a per-project convention. 🤷  So maybe it should be something like:

```suggestion
A portable Radicle identity solves this problem because you always own your Radicle identity. Your identity might be replicated across the Radicle network, but because your key signs it, it's always yours. Your project may be replicated across the Radicle network, but because your identity is a delegate, only you can change your own remote/branch.
```